ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerServices/LocalServerServices.cpp

```diff
@@ -24,7 +24,6 @@
 
 #include <QDesktopServices>
 #include <QDir>
-#include <QTemporaryDir>
```

Comment:
Not using `QTemporaryDir` now also has the unfortunate side effect of leaving around all these temporary directories. Is there a way to handle that?